citation/models.py

```diff
@@ -502,6 +502,22 @@ def primary(self, prefetch=False, **kwargs):
 
         return primary_pub
 
+    def aggregated_list(self, identifier=None, **kwargs):
+        """
+        :param: identifier: String - relation identifier - should be either sponsors, platforms, or container(i.e: journal)
+        :param: kwargs: Dict - additional query filter
+        :return: list of the aggregated data for the specified identifier
+        """
+        if identifier in ["sponsors", "platforms", "container"]:
+            return self.primary(status='REVIEWED', prefetch=True, **kwargs).annotate(
+                name=F(identifier + '__name')).values('name').order_by(
```

Comment:
Probably should document that the identifier that's passed in must be a Django Model with a `name` attribute.